services/uploads/serverless.yml

```diff
@@ -119,6 +119,54 @@ resources:
                   s3:ExistingObjectTag/virusScanStatus:
                     - "CLEAN"
                   aws:PrincipalArn: !GetAtt BucketAVScanRole.Arn
+            - Action: "s3:PutObject"
+              Effect: Deny
+              Principal: "*"
+              Resource:
```

Comment:
did you test this? based on this PR 
https://github.com/CMSgov/macstack-spa-submission-form/pull/227/files
You should only need the NotResource block starting on line 146